apps/web/pages/api/logo.ts

```diff
@@ -118,7 +118,7 @@ async function getTeamLogos(subdomain: string) {
     }
     // load from DB
     const { default: prisma } = await import("@calcom/prisma");
-    const team = await prisma.team.findUnique({
+    const team = await prisma.team.findFirst({
       where: {
         slug: subdomain,
       },
```

Comment:
Cant we get an overlap of slug being passed in here? 'Marketing' can exists in two or more places